deployment/docker-compose/README.md

```diff
@@ -0,0 +1,458 @@
+# Black Duck Alert On Docker Compose
+
+This document describes how to install and upgrade Alert in Docker Compose.
+Black Duck will no longer be supporting installations via Docker Compose in December 2019.  
+This installation method is deprecated and will not be supported after December 2019.
+
+## Table Of Contents
+- [Requirements](#requirements)
+- [Installing Alert](#installing-alert)
+    - [Standalone Installation](#standalone-installation)
+    - [Installation With Black Duck](#installation-with-black-duck)
+- [Upgrading Alert](#upgrading-alert)
+    - [Standalone Upgrade](#standalone-upgrade)
+    - [Upgrade With Black Duck](#upgrade-with-black-duck)
+- [Certificates](#certificates)
+- [Environment Variables](#environment-variables) 
+    - [Edit Environment File](#edit-environment-file)
+    - [Edit the Overrides File](#edit-the-overrides-file)
+    - [Environment Variable Overrides](#environment-variable-overrides)
+    - [Alert Hostname Variable](#alert-hostname-variable)
+    - [Alert Logging Level Variable](#alert-logging-level-variable)
+    - [Email Channel Environment Variables](#email-channel-environment-variables)
+    - [Environment Variable Classifications](#environment-variable-classifications)
+- [Advanced Configuration](#advanced-configuration)
+    - [Changing Server Port](#changing-server-port)
+    - [Changing Memory Settings](#changing-memory-settings)
+
+## Requirements
+
+- You have a Docker host with at least 2GB of allocatable memory.
+- You have administrative access to your docker host.  
+
+## Installing Alert
+Deployment files for Docker Compose are located in the docker-compose directory of the: 
```

Comment:
Colon seems a little strange here as I would be expecting a list but there is only a single value.